src/content/docs/zh/developers/guides/estimating-gas-and-tx-fees.mdx

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
no translation?

Reply:
I didn't see this page show up in local development and online docs in English. And I check the Spanish version, they also remains untransalted.
I guess it's a page working in progress?